tests/test_bond_assignment.py

```diff
@@ -1,5 +1,6 @@
 from autode import bond_lengths
 from autode.atoms import Atom
+import pytest
```

Comment:
as far as I can tell this is never used

Reply:
nope – thanks. will remove it in the next round 👍